apps/devhub/tests/test_utils.py

```diff
@@ -0,0 +1,373 @@
+import os.path
+from copy import deepcopy
+
+import mock
+import pytest
+from mock import call
+
+from django.conf import settings
+
+import amo
+import amo.tests
+from addons.models import Addon
+from files.models import File, FileUpload
+from versions.models import Version
+from devhub import utils
+from devhub.tasks import annotate_validation_results
+
+
+pytestmark = pytest.mark.django_db
```

Comment:
You don't need this here, because all the test cases are inheriting from `amo.tests.TestCase` which in turn inherits from Django's base test case. And `pytest-django` is clever enough to not impose the "database restriction" on those.